templates/openstack/pricing-calculator.html

```diff
@@ -0,0 +1,108 @@
+{% extends "openstack/_base_openstack.html" %}
+
+{% block title %}Private cloud pricing calculator{% endblock %}
+
+{% block meta_description %}Use the following private cloud pricing calculator to estimate costs of running your instances in the private cloud.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1n3ExZwFVlw5LyAp7O3CgPthVgx8yRX5-Wg_gBQLf73c/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h1>
+        Private cloud pricing
+      </h1>
+      <p>
+        While leading public cloud providers continue lowering their prices, private clouds remain a cost-efficient extension to the public cloud infrastructure. Get the following report for free to learn how cloud pricing differs across leading public and private cloud platforms, including AWS, Azure, Google, VMware and OpenStack.
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Private cloud pricing calculator</h2>
+  </div>
+  <div class="row">
+    <div class="col-7">
+      <h3 class="p-heading--4">Should your workloads run in a public or private cloud?</h3>
+      <p>Depending on your requirements, overall CapEx and OpEx costs and cost-per-resource metrics, it may be more economical to run your workloads in the private cloud. Use the following private cloud pricing calculator to estimate the costs of running your instances in the <a href="/openstack">Charmed OpenStack</a> private cloud. And see how much you can save compared to the public clouds.</p>
+    </div>
+    <div class="col-5 p-card">
+      <p class="p-heading--5">Have non-standard requirements?</p>
+      <p>Tell us more about your specific needs here - we will send you a detailed report and help you find the cloud architecture that best suits your requirements.</p>
+      <a class="js-invoke-modal p-button--positive" href="/openstack/contact-us">Get cost estimates</a>
+    </div>
+  </div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator" style="margin-bottom: 2rem;">
+  </div>
+
+  <div id="cost-calculator-section"></div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator" style="margin-bottom: 2rem; margin-top: 2rem;">
+  </div>
+
+  <div class="u-fixed-width">
+    <p><small>NOTE: The private cloud pricing calculator is for informational purposes only. Exact prices may vary depending on your detailed requirements, cloud configuration and local cost conditions. The hourly cost per instance contains all CapEx and OpEx costs over a three-year period, including reference hardware, typical hosting charges, average operations staff salary costs, cloud delivery and support subscriptions. The total savings are calculated compared to the reference AWS EC2 pricing for on-demand t3a type instances. In case of any questions, get in touch with our experts.</small></p>
```

Comment:
@anasereijo would this be better as a [information notification](https://vanillaframework.io/docs/patterns/notification#information)?

For example: 
![image](https://user-images.githubusercontent.com/1413534/125773374-4e6379a0-ef94-4bb6-8dfc-1d78ac626689.png)

Reply:
excellent idea